geolocator_android/android/src/main/java/com/baseflow/geolocator/LocationServiceHandlerImpl.java

```diff
@@ -59,6 +66,7 @@ public void onListen(Object arguments, EventChannel.EventSink events) {
 
     @Override
     public void onCancel(Object arguments) {
+        if(activity == null) return;
```

Comment:
Please add braces around the body of the `if` statement. With our plugins we try to adhere to the same [conventions](https://google.github.io/styleguide/javaguide.html#s4.1-braces) used by the first-party Flutter plugins.
```suggestion
        if (activity == null) {
          return;
        }
```